app.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Kitura - App for macOS</title>
+    <link rel="icon" type="image/png" href="assets/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="assets/favicon-16x16.png" sizes="16x16" />
+    <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" media="screen and (max-width: 900px)" href="css/mobile.css">
+  </head>
+  <header>
+    <div class="header-main">
+      <a class="home-link" href="index.html">
+        <img class="header-logo" src="assets/kitura-logo.png" alt="Kitura logo">
+        <h1 class="header-title">KITURA</h1>
+      </a>
+    </div>
+    <nav class="header-nav">
+      <a class="header-link" href="learn.html">LEARN</a>
+      <a class="header-link" href="packages.html">PACKAGES</a>
+      <a class="header-link" href="events.html">EVENTS</a>
+      <a class="header-link" href="help.html">HELP</a>
+    </nav>
+  </header>
+  
+  <body>
+    <div class="app-container">
+      <div class="app-left-container">
+        <img src="assets/mainscreen.png" alt="Kitura for macOS" width="500px">
+      </div>
+      <div class="app-right-container">
+        <h2>Kitura - macOS App</h2>
+        <h4>Get started with Kitura quickly using our new native macOS app. </h4>
+        <h4>There's no need to use the command line as the whole process, from package resolution to Xcode project generation, is automated.</h4>
+        <button class="button download-button" type="button" name="download-button" onclick="window.open('kitura.dmg')"><img class="slack-icon" src="assets/download.png" alt="Download Icon">DOWNLOAD</button>
+      </div> 
+    </div>
+    <div class="app-preview-container">
+      <div class="preview-heading-section">
+        <h2 style="color: white;">PREVIEW</h2>
```

Comment:
I don't think you need to use inline styling here. 

Couldn't you just do: 
```
.preview-heading-section {
    color: white;
}
```

And if you already styling for that class, just add 'color: white' to it?